rinja/Cargo.toml

```diff
@@ -39,7 +38,6 @@ with-warp = ["rinja_derive/with-warp"]
 [dependencies]
 rinja_derive = { version = "=0.3.5", path = "../rinja_derive" }
 
-humansize = { version = "2", optional = true }
```

Comment:
I think removing a feature is a breaking change. Maybe it would be better to keep it as a noop for now?

Reply:
We can make the next release a major one. Problem solved like that. :)

Reply:
Works for me! :D